### DISCUSSION TITLE
LDM Notes for December 9th, 2024

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#8895] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-12-09.md

### Agenda

- First-class span open questions
- Simple Lambda Parameters
- Collection expression arguments

**Comment by TonyValenti** [comment]
WRT collection expressions, perhaps consider "using" or "where" instead of "with".

**Reply by jnm2** [reply]
Some discussion on `where`: https://github.com/dotnet/csharplang/discussions/8886#discussioncomment-11566647
And `using`: https://github.com/dotnet/csharplang/discussions/8886#discussioncomment-11613725
